guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Hammer-cli.adoc

```diff
@@ -0,0 +1,56 @@
+[id="Using_{FreeIPA}_credentials_to_log_in_to_the_{Project}_Hammer_CLI_{context}"]
```

Comment:
```suggestion
[id="Using_{FreeIPA-context}_credentials_to_log_in_to_the_{Project}_Hammer_CLI_{context}"]
```

And also you'll have to define `:FreeIPA-context:` as base attribute and any other build target that overwrites "FreeIPA", e.g. satellite. This is necessary because satellte builds contain whitespace in `{FreeIPA}`.